src/components/QuoteApi.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+
+const apiKey = new Headers();
+apiKey.append('X-Api-Key', 'fi23b02FxA5RuBZcQMwheA==1pRuDF9DyRGAqaht');
+const apiSite = 'https://api.api-ninjas.com/v1/quotes?category=happiness';
+
+export default function QuoteApi() {
+  const [quote, setQuote] = useState([]);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  useEffect(() => {
+    const fetchQuotes = async () => {
+      try {
+        const response = await fetch(apiSite, {
+          method: 'GET',
+          headers: apiKey,
+        });
+
+        if (!response.ok) {
+          throw new Error('Network response was not ok.');
```

Comment:
- Optional
Instead of throwing the error, you can use the `setErrorMessage` setter to set the error and then display that error on the screen.